recipes/nested-dask/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nested-dask" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nested_dask-{{ version }}.tar.gz
+  sha256: f6a58439aaef48d5640f5a3f1d460d1a274d4511f569013d4839e30a054856d9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=62
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.9
+    - nested-pandas ==0.2.1
+    - numpy
+    - dask-core >=2024.3.0
+    - dask-expr
+    - pyarrow
+
+test:
+  imports:
+    - nested_dask
+  commands:
+    - pip check
+    - pytest tests
```

Comment:
We do not run full test suites in CI unless the build is particularly finicky. As this is a noarch: python package, I doubt this is needed. Let's remove.